refresh_screen_ui.gs

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
Can you please elaborate how this UI will be used?

Reply:
Whenever the call for the insights is directed to the API
it will return a token and the user can call the API again with that token but if the callback function time exceeds which is 25-30 seconds it will show this UI